daemon/info.go

```diff
@@ -87,6 +89,12 @@ func (daemon *Daemon) CmdInfo(job *engine.Job) engine.Status {
 	v.SetInt("NCPU", runtime.NumCPU())
 	v.SetInt64("MemTotal", meminfo.MemTotal)
 	v.Set("DockerRootDir", daemon.Config().Root)
+	if http_proxy != "" || https_proxy != "" {
+		v.Set("HttpProxy", http_proxy)
+		v.Set("HttpsProxy", https_proxy)
+		v.Set("NoProxy", os.Getenv("no_proxy"))
```

Comment:
I think we should `v.Set` each individually, only if the corresponding value is non-empty. IOW:

```
if httpProxy := os.Getenv("http_proxy"); httpProxy != "" {
    v.Set("HttpProxy", httpProxy)
}
// same for httpsProxy and noProxy
```

Note the camelcase change :)